client/rpc-api/src/offchain/mod.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017-2020 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Substrate offchain API.
+
+pub mod error;
+
+use jsonrpc_derive::rpc;
+use self::error::Result;
+use sp_core::offchain::StorageKind;
+
+pub use self::gen_client::Client as OffchainClient;
+
+/// Substrate offchain RPC API
+#[rpc]
+pub trait OffchainApi {
+	/// Set offchain local storage under given key and prefix.
+	#[rpc(name = "offchain_localStorageSet")]
+	fn set_local_storage(&self, kind: StorageKind, key: Vec<u8>, value: Vec<u8>) -> Result<()>;
```

Comment:
This all should be `Bytes` instead of `Vec<u8>` otherwise we will get an array of numbers instead of hex-string.